backend/src/authentication/mod.rs

```diff
@@ -335,8 +405,115 @@ mod tests {
         assert_eq!(response.status(), 200);
     }
 
+    #[test(sqlx::test(fixtures("../../fixtures/users.sql")))]
+    async fn test_whoami(pool: SqlitePool) {
+        let app = create_app(pool);
+
+        let response = app
+            .clone()
+            .oneshot(
+                Request::builder()
+                    .method(Method::POST)
+                    .uri("/api/user/login")
+                    .header(CONTENT_TYPE, "application/json")
+                    .body(Body::from(
+                        serde_json::to_vec(&Credentials {
+                            username: "user".to_string(),
+                            password: "password".to_string(),
+                        })
+                        .unwrap(),
+                    ))
+                    .unwrap(),
+            )
+            .await
+            .unwrap();
+
+        assert_eq!(response.status(), 200);
```

Comment:
```suggestion
        assert_eq!(response.status(), StatusCode::OK);
```
In https://github.com/kiesraad/abacus/pull/921 I converted all `response.status()` asserts to use `StatusCode::` instead of integers. Can you change the asserts you changed to use `StatusCode` as well? It's more explanatory than 401, 200, etc I think.

Reply:
Yes - fixed in 3becc157d68376cf790cd47c9ca84f740a4b892b